src/mono/netcore/System.Private.CoreLib/src/System/RuntimeType.Mono.cs

```diff
@@ -50,6 +50,8 @@ internal partial class RuntimeType
     {
         #region Definitions
 
+        internal IntPtr m_handle;
+
```

Comment:
Under mono, Type.Mono.cs contains this field already.

Reply:
I'm sorry Zoltan, I didn't find it, could you help me?

Reply:
https://github.com/dotnet/runtime/blob/b749e18bb16af4590753577601932dc2b5c67754/src/mono/netcore/System.Private.CoreLib/src/System/Type.Mono.cs#L14

I assume this is what he means?